Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1762,22 +1762,34 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const position = calculateSpaceAround(reference);
 
         const isVisibe = LayoutUtility.IsVisible(popUp);
+        let scrollbarGutterValue;
         if (!isVisibe)
         {
             popUp.style.width = 'auto'; // clear the width before showing the popUp, otherwise, the popUp expands to 100%
+            scrollbarGutterValue = popUp.style.getPropertyValue("scrollbar-gutter");
```

Comment:
naming: backups should be named `...Backup`. That way, you immidately get what this variable is intended for and how it should be used / if there is a usage missing / incorrect.
```suggestion
            scrollbarGutterValueBackup = popUp.style.getPropertyValue("scrollbar-gutter");
```